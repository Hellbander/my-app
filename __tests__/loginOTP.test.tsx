import '@testing-library/jest-dom'
import React from 'react'
import { fireEvent, render, screen } from '@testing-library/react'
import { useRouter } from 'next/router'
import LoginOTP from '@/app/otp/page'

jest.mock('next/navigation', () => ({
  useRouter() {
    return {
      prefetch: () => null
    }
  }
}))

describe('Login page', () => {
  it('renders login page', () => {
    const { container } = render(<LoginOTP />)
    expect(container).toMatchSnapshot()
  })
})