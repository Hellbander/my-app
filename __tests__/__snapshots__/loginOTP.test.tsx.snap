// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login page renders login page 1`] = `
<div>
  <main
    class="h-full flex justify-center align-middle"
  >
    <div
      class="flex flex-col justify-center align-middle px-6 py-12 lg:px-8"
    >
      <h2
        class="mb-10 text-2xl font-bold leading-9 text-gray-900"
      >
        OTP authenticator
      </h2>
      <div
        class="mt-10"
      >
        <button
          class="flex w-full justify-center rounded-md bg-blue-600 px-3 py-1.5 text-sm font-semibold leading-6 text-white shadow-sm hover:bg-blue-500 focus-visible:outline focus-visible:outline-2 focus-visible:outline-offset-2 focus-visible:outline-blue-600"
        >
          Send Pin
        </button>
      </div>
    </div>
  </main>
</div>
`;
